Mel test

ClimberMel opened the pull request:
Add module yf.py for fetching historical data using yFinance.  Module yfHist.py is an example of calling that functionality.

nagabrahmam-mantha commented:
Lets work with the yf2.py file (the one with classes). We can delete the other file yf.py (the one without classes)

Some questions/comments:
1. Under 'get_historic_stock_data' function, I think we can delete 'prepost', 'auto_adjust' etc parameters. You have them, but didnt use them anywhere.
2. For the parameters that you are passing, we need to synchronize them with IB and TD. Might be a tricky task, but we can come up with a solution.
3. The output for the yahoo data is a dataframe. I think its better to return json to keep it consistent with the rest.
4. For the 'get_historic_option_data', you can put "pass" under the function. We can fill it later when you have the code.

ClimberMel commented:
I have cleaned up the yf.py / yf2.py and now just have yf.py with working class structure

"delete 'prepost', 'auto_adjust' etc parameters. You have them, but didnt use them anywhere."
They are option parameters for yFinance, but I agree that I don't see ever using them so I have taken them out.

"For the parameters that you are passing, we need to synchronize them with IB and TD. Might be a tricky task, but we can come up with a solution."

I have lined up the parameters with the TDA API calls.  In TDData we should add the optiona start and end since the API allows for them and then the TDData and the YFData classes will match.
I'll work on the IBData class next.

I have changed the output for the yahoo data from a dataframe to json. 

For the 'get_historic_option_data', have replaced the code with "pass" under the function.  We can fill it later when you have the code.

ClimberMel commented:
Let me know what still needs to be done for some of these merges.
